front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/tables/index.ts

```diff
@@ -114,6 +114,9 @@ import { apiError } from "@app/logger/withlogging";
  *                 items:
  *                   type: string
  *                 description: Parent tables of this table
+ *               mime_type:
```

Comment:
🙏

Reply:
Not your code, but would be nice to change line 126 to reference the Table object instead of Datasource